docs/GHA.md

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions
+
+The Buildkite Migration tool supports transitioning GitHub Action workflow definitions to Buildkite pipelines.
```

Comment:
This is specified in the readme, I would change this sentence to be more an introduction of what to expect from this document (more details about what kind of things they should not expect to be translated)

Reply:
Tweaked with the latest!